x-pack/plugins/security_solution/common/api/detection_engine/rule_management/import_rules/rule_to_import.ts

```diff
@@ -29,5 +30,6 @@ export const RuleToImport = BaseCreateProps.and(TypeSpecificCreateProps).and(
   ResponseFields.partial().extend({
     rule_id: RuleSignatureId,
     immutable: z.literal(false).default(false),
+    required_fields: z.array(RequiredFieldInput).optional(),
```

Comment:
`required_fields` should be already in the structure; no need to redefine it

Reply:
Here I am overriding because `ResponseFields` contains `RequiredFieldArray` (with "ecs") and as I understand for import `RequiredFieldInput` (no "ecs") is enough.